torch/csrc/jit/tensorexpr/bounds_overlap.h

```diff
@@ -78,6 +78,14 @@ OverlapKind TORCH_API overlaps(const IndexBounds& a, const IndexBounds& b);
 // Multiple Bounds can be returned in the case where B slices A into two
 // distinct regions with no overlap.
 //
+// For example:
+//    subtractBound((0, 10), (2, 4)) => [(0, 1), (5, 10)]
+//       bound A: (0, 10)
+//       bound B: (2, 4)
+//       If we remove slice (2, 4) from the slice (0, 10), we will be left
+//       with 2 slices, one at the start (0, 1), and one at the end (5, 10).
+//       So, the result of this subtraction is [(0, 1), (5, 10)].
```

Comment:
That's super helpful! I realized that I completely misunderstood the function, I thought we were figuring out a boundary for `i - j` expression given boundaries for `i` and `j`.